packages/terra-functional-testing/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Fixed
   * Update specPath in BaseCompare to replace `node_modules` with `tests/wdio`.
 
+* Added
+  * Added useSeleniumStandalonService option for using the standalone-chrome host instead of the selenium docker service when building in Jenkins.
```

Comment:
```suggestion
  * Added useSeleniumStandaloneService option for using the standalone-chrome host instead of the selenium docker service when building in Jenkins.
```

Reply:
Good catch. Updated all.
https://github.com/cerner/terra-toolkit/pull/646/commits/ee6355ab0f9f8b345db66ce3900222f0f3d87731